cmake/modules/FindRif.cmake

```diff
@@ -15,7 +15,7 @@ endmacro(SET_RIF_VARIABLES)
 if(APPLE)
     SET_RIF_VARIABLES(OSX)
 elseif(WIN32)
-    SET_RIF_VARIABLES(Windows)
+    SET_RIF_VARIABLES(Windows/Dynamic)
```

Comment:
Add this inside of `SET_RIF_VARIABLES` to fix compilation on all platforms, not just on windows

Reply:
Done